docs/strategies/circuit-breaker.md

```diff
@@ -91,23 +94,29 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddCircuitBreaker(optionsSt
+|--------------------------|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ShouldHandle`           | Any exceptions other than `OperationCanceledException`. | Defines a predicate to determine what results and/or exceptions are handled by the circuit breaker strategy.                                              |
+| `FailureRatio`           | 0.1                                                     | The failure-success ratio that will cause the circuit to break/open. `0.1` means 10% failed of all sampled executions.                                    |
+| `MinimumThroughput`      | 100                                                     | The minimum number of actions that must occur within the specified sampling duration.                                                                     |
+| `SamplingDuration`       | 30 seconds                                              | The time period over which the failure-success ratio is calculated.                                                                                       |
+| `BreakDuration`          | 5 seconds                                               | Defines a **static** time period for which the circuit will remain broken/open before attempting to reset.                                                |
+| `BreakDurationGenerator` | `null`                                                  | This delegate allows you to **dynamically** calculate the break duration by utilizing information that is only available at runtime (like failure count). |
+| `ManualControl`          | `null`                                                  | If provided then the circuit's state can be manual controlled via a `CircuitBreakerManualControl` object.                                                 |
+| `StateProvider`          | `null`                                                  | If provided then the circuit's current state can be retrieved via a `CircuitBreakerStateProvider` object.                                                 |
+| `OnClosed`               | `null`                                                  | If provided then it will be invoked after the circuit transitioned either to the `Closed` or to the `Isolated` state.                                     |
+| `OnOpened`               | `null`                                                  | If provided then it will be invoked after the circuit transitioned to the `Opened` state.                                                                 |
+| `OnHalfOpened`           | `null`                                                  | If provided then it will be invoked after the circuit transitioned to the `HalfOpened` state.                                                             |
```

Comment:
```suggestion
| `OnClosed`               | `null`                                                  | If provided then it will be invoked after the circuit transitions to either the `Closed` or `Isolated` states.                                     |
| `OnOpened`               | `null`                                                  | If provided then it will be invoked after the circuit transitions to the `Opened` state.                                                                 |
| `OnHalfOpened`           | `null`                                                  | If provided then it will be invoked after the circuit transitions to the `HalfOpened` state.                                                             |
```